web-src/src/components/PromptTemplateLibraryPanel.js

```diff
@@ -54,7 +54,7 @@ export function PromptTemplateLibraryPanel({ props }) {
     if (selectedTemplate.isNew) {
       sampleRUM('genai:prompt:new', { source: 'HomePanel#handleSelect' });
     } else {
-      sampleRUM(`genai:prompt:${selectedTemplate.isAdobe ? 'isAdobe' : 'isCustom'}`, { source: 'HomePanel#handleSelect' });
+      sampleRUM(`genai:prompt:${selectedTemplate.isAdobe ? 'isadobe' : 'iscustom'}`, { source: 'HomePanel#handleSelect' });
```

Comment:
Since you’re on it. Could we rename it to ‘isadobeselected’ and ‘iscustomselected’?